### ISSUE TITLE
Accessibility: Back button in all setting screens is not labeled

### ISSUE THREAD

**Issue by beqabeqa473** [signalapp/Signal-Android#11498]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [ ] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Back button in all setting screens have not label

### Steps to reproduce
- open settings and focus back button on top left corner of screen
**Actual result:** talkback will say unlabeled button
**Expected result:** talkback should correctly read label on this button

### Device info

**Device:** Samsung Galaxy s21
**Android version:** 11.0.0
**Signal version:** Latest

**Comment by alex-signal** [comment]
Could you please confirm whether this is also the case for other instances, such as the "new conversation" screen? I'd like to make sure we know going forward what needs a manually set `navigationContentDescription` and what doesn't (the field that should be pulled by talkback)

**Comment by beqabeqa473** [comment]
No, there is not a case with new conversation screen.

Talkback says "Navigate up"

On 8/2/21, Alex Hart ***@***.***> wrote:
> Could you please confirm whether this is also the case for other instances,
> such as the "new conversation" screen? I'd like to make sure we know going
> forward what needs a manually set `navigationContentDescription` and what
> doesn't (the field that should be pulled by talkback)
>
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/signalapp/Signal-Android/issues/11498#issuecomment-891023759


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili

**Comment by alex-signal** [comment]
Thanks for the response! I'll make sure this is updated to match.
